modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -358,13 +358,11 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
             ///////////////////////////////////////////////////////////////////////
             ////////////// Make sure the file name matches the format   ///////////
             ///////////////////////////////////////////////////////////////////////
-            if (($file_name != $pscid."_". $candid ."_". $visit_label . ".zip")
-                && ($file_name != $pscid."_". $candid ."_". $visit_label . ".tgz")
-                && ($file_name != $pscid."_". $candid ."_". $visit_label . ".tar.gz")
-            ) {
```

Comment:
Suggestion:
```
/**
   * Checks if the inputted file name is valid
   *
   * @param {string} requiredFileName - Required file name
   * @param {string} fileName - Provided file name
   * @return {boolean} - true if file names match and false otherwise
   */
  isValidFileName: function(requiredFileName, fileName) {
    if (fileName === null || requiredFileName === null) {
      return false;
    }

    return (fileName.indexOf(requiredFileName) > -1);
  }
```